packages/components/text/README.md

```diff
@@ -2,4 +2,15 @@
 
 A cross-platform Text component using the Fluent Design System.
 
-Text V1 supported platforms: win32
\ No newline at end of file
+Text elements can be specified either by passing a `variant` [typography token](../../../docs/pages/Theming/Tokens/Basics.md#typography) into a `TextV1` element or by using predefined JSX types. For example:
```

Comment:
This information may be covered in the SPEC.md file already. If so we could link to the SPEC.md file here for usage guidance.

Reply:
Done.